packages/block-editor/src/components/rich-text/format-toolbar/index.js

```diff
@@ -11,11 +11,11 @@ import { orderBy } from 'lodash';
 import { __ } from '@wordpress/i18n';
 import { Toolbar, Slot, DropdownMenu } from '@wordpress/components';
 
-const FormatToolbar = ( { controls } ) => {
+const FormatToolbar = () => {
 	return (
 		<div className="editor-format-toolbar block-editor-format-toolbar">
 			<Toolbar>
-				{ controls.map( ( format ) =>
+				{ [ 'bold', 'italic', 'link' ].map( ( format ) =>
```

Comment:
Forgot to ask, why is this hardcoded now?

Reply:
Ah, good question! These names are _kind of_ hardcoded already in the formatting library. You cannot really add anything else here. But I forgot about the case where you'd remove the controls by setting an empty array. That case was already only partially working in `master`, see #14542. Let's try to correct it there again.

Reply:
Now fixed in #14542.